content/es/devops.md

```diff
@@ -0,0 +1,16 @@
+---
+title: DevOps
+status: Completed
+category: concept
+---
+
+## Qué es
+DevOps es una metodología en la que los equipos son dueños de todo el proceso, desde el desarrollo de la aplicación hasta las operaciones en producción, de ahí el término DevOps. Va más allá de implementar un conjunto de tecnologías y requiere un cambio completo en la cultura y los procesos. DevOps requiere grupos de ingenieros que trabajen en componentes pequeños (en lugar de una función completa), reduciendo las transferencias, una fuente común de errores.
+
+## Problema que aborda
+Tradicionalmente, en organizaciones complejas con [aplicaciones monolíticas](/monolithic_apps/) y arquitecturas con [alto acoplamiento](/tightly_coupled_architectures/), el trabajo generalmente se distribuía entre varios equipos. Esto dio lugar a numerosos traspasos y largos plazos de entrega. Cada vez que un componente o alguna actualización estaba lista, se colocaba en una cola para el siguiente equipo. Debido a que las personas solo trabajaron en una pequeña parte del proyecto, este enfoque condujo a una falta de propiedad. Su objetivo era entregar el trabajo al siguiente grupo, en vez de ofrecer la funcionalidad correcta al cliente - un claro desajuste de prioridades.
```

Comment:
```suggestion
Tradicionalmente, en organizaciones complejas con [aplicaciones monolíticas](/monolithic_apps/) con arquitecturas [estrechamente acopladas](/tightly_coupled_architectures/), el trabajo generalmente se distribuía entre varios equipos. Esta fragmentación del desarrollo dio lugar a numerosos traspasos y largos plazos de entrega. Cada vez que un componente o alguna actualización estaba lista, se colocaba en una cola para el siguiente equipo. Debido a que las personas solo trabajaron en una pequeña parte del proyecto, este enfoque condujo a una falta de propiedad. Su objetivo era entregar el trabajo al siguiente grupo, en vez de ofrecer la funcionalidad correcta al cliente - un claro desajuste de prioridades.
```

Reply:
https://nordicapis.com/the-difference-between-tight-coupling-and-loose-coupling/